test/unit/vl-upload.test.html

```diff
@@ -251,6 +260,16 @@
           assert.equal(upload.getTranslation('upload.add_files_subtitle'), upload.getAttribute('sub-title'));
         });
 
+        test('de titel en sub-titel kan gepersonaliseerd worden via een slot', () => {
+          const upload = fixture('vl-upload-slots-text-fixture');
+          awaitUntil(() => upload._titleSlotElement).then(() => {
+            assert.equal(upload._titleSlotElement.assignedNodes()[0].textContent, 'Titel');
+          });
+          awaitUntil(() => upload._subTitleSlotElement).then(() => {
+            assert.equal(upload._subTitleSlotElement.assignedNodes()[0].textContent, 'Sub-titel');
+          });
+        });
```

Comment:
Deze test is niet asynchroon en zal bijgevolg altijd groen zijn. Wanneer je hem asynchroon zou maken, zal je zien dat hij eigenlijk nooit groen zou kunnen zijn aangezien `_titleSlotElement` en `_subTitleSlotElement` verwijderd worden.